apps-rendering/src/components/RichLink/index.tsx

```diff
@@ -54,45 +58,30 @@ const liveBlogRichLinkStyles = css`
 `;
 
 const richLinkStyles = (format: ArticleFormat): SerializedStyles => {
-	const { kicker: newsKicker, inverted: newsInverted } = getThemeStyles(
-		ArticlePillar.News,
-	);
-	const { kicker: opinionKicker, inverted: opinionInverted } = getThemeStyles(
-		ArticlePillar.Opinion,
-	);
-	const { kicker: sportKicker, inverted: sportInverted } = getThemeStyles(
-		ArticlePillar.Sport,
-	);
-	const { kicker: cultureKicker, inverted: cultureInverted } = getThemeStyles(
-		ArticlePillar.Culture,
-	);
-	const { kicker: lifestyleKicker, inverted: lifestyleInverted } =
-		getThemeStyles(ArticlePillar.Lifestyle);
-
 	return css`
-		background: ${backgroundColor(format)};
+		background: ${background.richLink(format)};
 		padding: ${remSpace[3]} ${remSpace[3]} ${remSpace[2]};
-		border-top: solid 1px ${neutral[60]};
+		border-top: solid 1px ${border.richLinkDark(format)};
 		transition: all 0.2s ease;
 
 		&.js-news {
-			${richLinkPillarStyles(newsKicker, newsInverted)}
+			${richLinkPillarStyles(format)}
 		}
 
 		&.js-opinion {
-			${richLinkPillarStyles(opinionKicker, opinionInverted)}
+			${richLinkPillarStyles(format)}
 		}
 
 		&.js-sport {
-			${richLinkPillarStyles(sportKicker, sportInverted)}
+			${richLinkPillarStyles(format)}
 		}
 
 		&.js-culture {
-			${richLinkPillarStyles(cultureKicker, cultureInverted)}
+			${richLinkPillarStyles(format)}
 		}
 
 		&.js-lifestyle {
-			${richLinkPillarStyles(lifestyleKicker, lifestyleInverted)}
+			${richLinkPillarStyles(format)}
 		}
```

Comment:
Is `format` the `Format` of the main article? If so, how will rich links pick up the format of the articles they're pointing to?

Reply:
Ahh good shout! 🤦 
Fixed this.